examples/megacli.cpp

```diff
@@ -2571,7 +2571,7 @@ autocomplete::ACN autocompleteSyntax()
     p->Add(exec_rm, sequence(text("rm"), remoteFSPath(client, &cwd)));
     p->Add(exec_mv, sequence(text("mv"), remoteFSPath(client, &cwd, "src"), remoteFSPath(client, &cwd, "dst")));
     p->Add(exec_cp, sequence(text("cp"), remoteFSPath(client, &cwd, "src"), either(remoteFSPath(client, &cwd, "dst"), param("dstemail"))));
-    p->Add(exec_du, sequence(text("du"), remoteFSPath(client, &cwd)));
+    p->Add(exec_du, sequence(text("du"), either(sequence(flag("-all"), opt(flag("-breakdown"))), remoteFSPath(client, &cwd))));
```

Comment:
Does that mean a remote path cannot be called `-all`?

Reply:
correct: `-all` is done by iterating the map of all nodes, rather than recursing subtrees.